src/Tests/Microsoft.Diagnostics.Monitoring.UnitTests.ApiClient/Microsoft.Diagnostics.Monitoring.UnitTests.ApiClient.csproj

```diff
@@ -0,0 +1,26 @@
+<Project Sdk="Microsoft.NET.Sdk">
+
+  <PropertyGroup>
+    <TargetFramework>netcoreapp3.1</TargetFramework>
+  </PropertyGroup>
+
+  <ItemGroup>
+    <OpenApiReference Include="..\..\..\documentation\openapi.json" Namespace="Microsoft.Diagnostics.Monitoring.UnitTests" ClassName="ApiClient">
+      <CodeGenerator>NSwagCSharp</CodeGenerator>
+      <Link>OpenAPIs\openapi.json</Link>
+    </OpenApiReference>
+  </ItemGroup>
+
+  <ItemGroup>
+    <PackageReference Include="Microsoft.Extensions.ApiDescription.Client" Version="$(MicrosoftExtensionsApiDescriptionClientVersion)">
+      <PrivateAssets>all</PrivateAssets>
+      <IncludeAssets>runtime; build; native; contentfiles; analyzers; buildtransitive</IncludeAssets>
+    </PackageReference>
+    <PackageReference Include="Newtonsoft.Json" Version="$(NewtonsoftJsonVersion)" />
+    <PackageReference Include="NSwag.ApiDescription.Client" Version="$(NSwagApiDescriptionClientVersion)">
```

Comment:
While this is test only code that never gets shipped, do we need to update our 3rd party library files?

Reply:
No, as long as it's just testing assets it doesn't go in the Third Party Notices